src/main/java/com/palantir/gradle/gitversion/JGitDescribe.java

```diff
@@ -76,6 +77,8 @@ public String describe(String prefix) {
 
                 head = walk.parseCommit(parents[0]);
             }
+
+            walk.close();
```

Comment:
I don't think this is required but it's safe to close this walk.

Reply:
this is definitely not necessary, the try-with-resources block will call `walk.close()` automatically.